utils.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
@bfallik No disrespect was intended by my original comment. I will ask more questions in the future if anything is unclear, but would appreciate it if you would take the time to elaborate on your feedback during the first pass so as to avoid the wasted resources. Let me know if you would like to F2F to clear anything up.